notebooks/algorithm-design/concepts.ipynb

```diff
@@ -0,0 +1,490 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Basic concepts"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This course is meant to be an introduction to quantum variational algorithms. Before we dive into the specifics of these algorithms, there are some quantum computation concepts that are essential to understand how and why they work. Most of them are covered in [Understanding Quantum Information and Computation](https://qiskit.org/learn/course/basics-quantum-information/) (also available as [a series of YouTube videos](https://www.youtube.com/playlist?list=PLOFEBzvs-VvqKKMXX4vbi4EB1uaErFMSO)). We strongly recommend that you go through the contents of that course first, as we will assume some familiarity with the theoretical context it provides. However, in this chapter we offer a quick reminder of some of the most important concepts. After that, we introduce Qiskit's *primitives*, the building blocks of variational and many other quantum algorithms."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### General quantum systems\n",
```

Comment:
You could consider trimming each of these headings down and putting them in a tabs block (such as the one at the top of [_What is quantum?_](https://learn.qiskit.org/course/introduction/what-is-quantum)).